content/community/team.md

```diff
@@ -8,169 +8,18 @@ This page lists who we are. By all means add yourself to the list - lets sort it
-| Ioannis Canellos       | iocanel          | Red Hat                    |
-| Jacek Laskowski        | jlaskowski       |                            |
-| James Carman           | jcarman          | Savoir Technologies        |
-| Jan Bednar             | janbednar        | CGI IT Czech Republic      |
-| Jean-Baptiste Onofré   | jbonofre         | Talend                     |
-| Jeff Genender          | jgenender        | Savoir Technologies        |
-| Johan Edstrom          | joed             | Savoir Technologies        |
-| John Poth              |                  | Red Hat                    |
-| Jonathan Anstey        | janstey          | Red Hat                    |
-| James Netherton        | Red Hat          |                            |
-| James Strachan         | jstrachan        |                            |
-| Luca Burgazzoli        | lburgazzoli      | Red Hat                    |
-| Martin Krasser         | krasserm         |                            |
-| Nicky Sandhu           | nsandhu          |                            |
-| Onder Sezgin           | onders           |                            |
-| Omar Al-Safi           | oalsafi          | Talend                     |
-| Quinn Stevenson        | quinn            |                            |
-| Raul Kripalani         | raulk            |                            |
-| Pascal Schumacher      | pascalschumacher |                            |
-| Peter Palaga           | ppalaga          | Red Hat                    |
-| Rich Newcomb           | rnewcomb         |                            |
-| Richard Kettelerij     | rickette         | Avisi BV                   |
-| Rob Davies             | rajdavies        |                            |
-| Roman Kalukiewicz      | romkal           |                            |
-| Scott England-Sullivan | sully6768        | Red Hat                    |
-| Stan Lewis             | slewis           | Red Hat                    |
-| Tadayoshi Sato         | tsato            | Red Hat                    |
-| Tomohisa Igarashi      | igarashitm       | Red Hat                    |
-| Tracy Snell            | tjsnell          | Juice Labs                 |
-| Willem Jiang           | ningjiang        | Huawei                     |
-| William Tam            | wtam             | Red Hat                    |
-| Zoran Regvart          | zregvart         | Red Hat                    |
-{{< /table >}}
 
-## Contributors
+{{< committers >}}
+
+
+
 
+
+## Contributors
 Adding your name to the list below.
 
-If you have been contributing to the Apache Camel project, and you want your name added to the list below. Then you can get in touch with the Camel team from the [Mailing Lists](../mailing-list/) and ask to be added.
+If you have been contributing to the Apache Camel project, and you want your name added to the list below. Then you can get in touch with the Camel team from the <a href="../mailing-list/">Mailing Lists</a> and ask to be added.
```

Comment:
Leave the markup as it is. No need to add HTML markup for links here.

Reply:
fixed